src/lib.rs

```diff
@@ -2,7 +2,7 @@ extern crate core;
 
 mod app;
 mod app_main;
-mod bloom;
+// mod bloom;
```

Comment:
Should the bloom filter implementation be removed completely, or does it make sense to keep it?